src/views/CashlinkManage.vue

```diff
@@ -0,0 +1,342 @@
+                            </g>
+                        </svg>
+                    </a>
+                    <a class="nq-button-s social-share" :href="whatsapp" target="_blank">
+                        <svg width="28" height="28" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <path opacity=".6" d="M14 0C6.3 0 0 6.235 0 14c0 2.823.817 5.647 2.45 8L.817 26.706c-.117.235 0 .47.116.588.117.118.35.235.584.118l5.016-1.53C13.067 30 21.7 28 25.783 21.412c1.4-2.235 2.1-4.824 2.1-7.412C28 6.235 21.7 0 14 0zm8.4 19.647c-.817 1.53-2.333 2.47-4.083 2.47-1.4-.117-2.8-.588-3.967-1.176-2.8-1.294-5.25-3.412-7-6.117-2.217-2.942-2.333-5.765-.233-8.118.583-.47 1.4-.706 2.216-.588.584.117 1.167.47 1.4 1.058l.467 1.06c.35.823.7 1.646.7 1.764.233.353.233.823 0 1.177-.35.588-.7 1.176-1.167 1.646a10.68 10.68 0 002.1 2.471c.934.824 1.984 1.53 3.15 1.883.35-.471.934-1.177 1.167-1.53a1.175 1.175 0 011.517-.47c.466.117 2.916 1.411 2.916 1.411.35.118.584.353.817.588.467.942.467 1.765 0 2.471z"/>
+                        </svg>
+                    </a>
+                </template>
+            </PageFooter>
+        </SmallPage>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator';
+import staticStore, { Static } from '../lib/StaticStore';
+import { ParsedCashlinkRequest, RequestType } from '../lib/RequestTypes';
+import { SmallPage, PageBody, PageFooter, Account, CheckmarkSmallIcon, Copyable } from '@nimiq/vue-components';
+import StatusScreen from '../components/StatusScreen.vue';
+import { NetworkClient } from '@nimiq/network-client';
+import { loadNimiq } from '../lib/Helpers';
+import Cashlink from '../lib/Cashlink';
+import { CashlinkState } from '@/lib/PublicRequestTypes';
+import { CashlinkStore } from '../lib/CashlinkStore';
+import { State } from 'vuex-class';
+import KeyguardClient from '@nimiq/keyguard-client';
+import { Cashlink as PublicCashlink } from '../lib/PublicRequestTypes';
+import { Clipboard } from '@nimiq/utils';
+import Config from 'config';
+
+@Component({components: {
+    Account,
+    CheckmarkSmallIcon,
+    PageBody,
+    PageFooter,
+    SmallPage,
+    StatusScreen,
+    Copyable,
+}})
+export default class CashlinkManage extends Vue {
+    @Static private request!: ParsedCashlinkRequest;
+    @Static private cashlink!: Cashlink;
+    @Static private keyguardRequest!: KeyguardClient.SignTransactionRequest;
+    @State private keyguardResult?: KeyguardClient.SignTransactionResult;
+
+    private isTxSent: boolean = false;
+    private isManagementRequest = false;
+    private status: string = 'Connecting to network...';
+    private state: StatusScreen.State = StatusScreen.State.LOADING;
```

Comment:
The state (and thus also the title) only get changed after `isTxSent` switches to true which hides the status screen. Thus, `state` and `title` can be completely removed.

Reply:
Switched out the order in which `state` and `isTxSend` is updated. Now it makes more sense.

Reply:
With the new changes, the success screen is displayed before getting to the management screen.
I think Julian stated at some point that the success screen should only be shown at the very end of a flow.
Please consult with him about this.

Therefore, the suggestion back then was that `state` and `title` can be completely removed.
However, with the Christmas cashlink changes, I actually need these again.
Thus, it's fine to keep these, but the behavior should be switched back to what it was previously, depending on what Julian says.